cvat-core/src/annotations-objects.ts

```diff
@@ -3085,35 +3114,39 @@ Object.defineProperty(EllipseTrack, 'distance', { value: EllipseShape.distance }
 Object.defineProperty(CuboidTrack, 'distance', { value: CuboidShape.distance });
 Object.defineProperty(SkeletonTrack, 'distance', { value: SkeletonShape.distance });
 
-export function shapeFactory(data: SerializedShape, clientID: number, injection: AnnotationInjection): Shape {
+export function shapeFactory(
+    data: SerializedShape | SerializedShape['elements'][0],
```

Comment:
No, it is not converted in case of PointsShape/PointsTrack, because points may be skeleton elements. And they have corresponding constructors, that may be initialized with skeleton element data